### PR TITLE
agregado enviar mensaje de c++ a java

### PR DESCRIPTION
**Pull request by luisandia**

### CODE REVIEW

File: app/src/main/cpp/vraze_app.cc [hunk 1]
```diff
@@ -82,6 +82,9 @@ VRazeApp::VRazeApp(JNIEnv *env, jobject asset_manager, jlong gvr_context_ptr)
       steering_rotation_(0.0f) {
   fplbase::SetAAssetManager(AAssetManager_fromJava(env, asset_manager));
   LOGD("VRazeApp initialized.");
+
+  this->env=env;
```
Comment (alculquicondor) [inline app/src/main/cpp/vraze_app.cc]: Use `env(env)` on initializers instead.

File: app/src/main/cpp/vraze_app.cc [hunk 2]
```diff
@@ -185,7 +188,9 @@ void VRazeApp::OnDrawFrame() {
 
   GetInput();
   car_physics_->Move(delta_time, accelerating_, braking_, steering_rotation_);
-
+  //SendMessage(head_view_gvr.m[0][0],head_view_gvr.m[0][1],head_view_gvr.m[1][0],head_view_gvr.m[1][1]);
```
Comment (alculquicondor) [inline app/src/main/cpp/vraze_app.cc]: remove this comment

File: app/src/main/cpp/vraze_app.cc [hunk 3]
```diff
@@ -258,3 +263,33 @@ void VRazeApp::SetUpViewPortAndScissor(const gvr::Sizei &framebuf_size,
   // TODO: Discover how to make this work
   // renderer_->ScissorOn({left, bottom}, {width, height});
 }
+
+void VRazeApp::SendMessage(float x,float y, float dir0,float dir1){
+  // int status=this->vm->GetEnv((void**)&env,JNI_VERSION_1_6);
```
Comment (alculquicondor) [inline app/src/main/cpp/vraze_app.cc]: remove unused code

File: app/src/main/cpp/vraze_app.cc [hunk 4]
```diff
@@ -258,3 +263,33 @@ void VRazeApp::SetUpViewPortAndScissor(const gvr::Sizei &framebuf_size,
...
+  // First get the class that contains the method you need to call
+  jclass clazz;// = env->FindClass("pe/edu/ucsp/vraze/MainActivity");
+
+  clazz = (jclass) env->NewGlobalRef(env->FindClass("pe/edu/ucsp/vraze/MainActivity"));
+  if(!clazz){
+    LOGD("HORROR - CANNOT FIND CLASS");
+  }
+
+  // Get the method that you want to call "(Ljava/lang/String;)V"
+  jmethodID metodo = env->GetStaticMethodID(clazz, "messageMe", "(FFFF)V");
```
Comment (alculquicondor) [inline app/src/main/cpp/vraze_app.cc]: you can get this ID on the constructor, store it in a class member and then just call `CallStaticVoidMethod` here

File: app/src/main/cpp/vraze_app.h [hunk 5]
```diff
@@ -67,6 +68,8 @@ class VRazeApp {
   bool accelerating_;
   bool braking_;
 
+  JNIEnv *env;
```
Comment (alculquicondor) [inline app/src/main/cpp/vraze_app.h]: name it `env_`

File: app/src/main/cpp/vraze_app.h [hunk 6]
```diff
@@ -67,6 +68,8 @@ class VRazeApp {
...
+  JavaVM* vm;
```
Comment (alculquicondor) [inline app/src/main/cpp/vraze_app.h]: name it `vm_`

File: app/src/main/java/pe/edu/ucsp/vraze/MainActivity.java [hunk 7]
```diff
@@ -60,6 +60,12 @@ public void run() {
         }
       };
 
+  public static void messageMe(float x, float y, float dir0, float dir1) {
+    System.out.println("x: " + String.valueOf(x) + " y: " + String.valueOf(y));
+    System.out.println("d0: " + String.valueOf(dir0) + " d1: " + String.valueOf(dir1));
+    //Log.d(TAG, Arrays.toString(text));
```
Comment (alculquicondor) [inline app/src/main/java/pe/edu/ucsp/vraze/MainActivity.java]: Why println instead of log?

**Review by alculquicondor — CHANGES_REQUESTED** [review]

File: app/src/main/java/pe/edu/ucsp/vraze/MainActivity.java (hunk 7 above)
Comment (luisandia) [reply app/src/main/java/pe/edu/ucsp/vraze/MainActivity.java]: For nothing in special, i'll fix it

**Review by alculquicondor — APPROVED** [review]

**Pull request merged**